CONTRIBUTING.md

```diff
@@ -56,6 +56,15 @@ If you want to run the website version locally, run:
 (.env)$ python app.py
 ```
 
+If you want to run the website locally, but would prefer to use Docker instead of installing python, you can build a container and run it like so:
+```bash
+docker build -t NAME_OF_CONTAINER .
```

Comment:
technically it's the name of the image not the name of the container. I would instead just name the image `hedy` instead.

Reply:
this is much simpler, and very sensible. thanks!